src/classes/ContactTriggerHandler.cls

```diff
@@ -0,0 +1,47 @@
+/**
+* Description   - Contact Trigger Handler
+* Created by    - Sagar Nirwal
+* Created Date  - 028/08/2022
+* Revision log  -  V_1.0 - Created  - Sagar Nirwal - 28/08/2022
+*                  V_1.1 - Modified - Sagar NIrwal - 28/08/2022 -  Details what you modified
+**/
+public with sharing class ContactTriggerHandler {
+    /**   Description :   Method synchronize Record
+    *     Created By  :   Sagar Nirwal
+    *     Arguments   :   List<Contact>
+    *     Return      :   void
+    *     Revision log:   V_1.0 - Created  - Sagar Nirwal - 28/08/2022
+    *                     V_1.1 - Modified - Sagar Nirwal - 28/08/2022 - Details what you modified
+    **/
+    public static void synchronizeContactAccountPropertyRecord(List<Contact> newContactList) {
+        Set<Id> setOfAccountIds = new Set<Id>();
+        for(Contact con: newContactList){
+            if(con.Status__c != null || String.isBlank(con.Status__c)){
```

Comment:
If should be like if(con.Status__c == null && (oldMap == null || oldMap.get(con.Id).Status__c != con.Statuc__c))